lib/src/layer/polygon_layer/projected_polygon.dart

```diff
@@ -25,7 +25,11 @@ class _ProjectedPolygon {
           ),
           holePoints: () {
             final holes = polygon.holePointsList;
-            if (holes == null) return null;
+            if (holes == null ||
+                holes.isEmpty ||
+                holes.every((e) => e.isEmpty)) {
+              return <List<DoublePoint>>[];
+            }
```

Comment:
`holes.every()` is an expensive check.
It could be put into the list generation beneath.
Something like `holes.where((e) => e.isNotEmpty).map((e) => ...).toList(growable: false)` should do the same but more efficient.

Reply:
I'm not too fussed about this. It should only ever be expensive when the user specifies a list like so `[[], [], []]`, which there is no real reason to do. I've also put it last in the list to take advantage of the lazy logic gates.

Reply:
I see your point. In that case the performance penalty is not as much as I've thought.